src/framework/preferences/js/PrefsEditor.js

```diff
@@ -380,7 +380,7 @@ var fluid_2_0 = fluid_2_0 || {};
 
         var initialModel = that.initialModel,
             userSelections = fluid.copy(that.model),
-            changedSelections = {};
+            changedPrefs = {};
```

Comment:
The name seems wrong, if the changes are more than just what's stored under model.preferences.